pom.xml

```diff
@@ -195,17 +195,14 @@
         <dependency>
             <groupId>org.osgi</groupId>
             <artifactId>osgi.core</artifactId>
-            <version>6.0.0</version>
         </dependency>
         <dependency>
             <groupId>org.osgi</groupId>
             <artifactId>osgi.cmpn</artifactId>
-            <version>6.0.0</version>
         </dependency>
         <dependency>
             <groupId>org.osgi</groupId>
             <artifactId>org.osgi.service.event</artifactId>
```

Comment:
Please either depend on the individual OSGi artifacts (preferably) or the aggregates (osgi.cmpn) but not both, for details refer to https://github.com/apache/sling-parent/blob/35ff6b88ff7e69a9da0f94be1997b7a72450282e/sling-bundle-parent/pom.xml#L222

Reply:
Not sure it needs the whole thing - let me try without `osgi.cmpn` ..

Reply:
ok, test seems to work without this. 

However, I feel like this is going down a rabbit hole, and I'd prefer to continue for now and look at this later. 

@kwin, are you -1 on shipping with this included? The parent pom says "preferred", so that doesn't sound like a must.

Reply:
SLING-10302 created

Reply:
I am fine with tackling that later, but please remove `<artifactId>org.osgi.service.event</artifactId>` then as this 
a) is already contained in `osgi.cmpn`
b) comes later in the class path, so will never be considered by Maven